docs/data/routes/docs/fundamentals/services/dictionary/sitecore-dictionary-service/en.md

```diff
@@ -0,0 +1,27 @@
+---
+name: sitecore-dictionary-service
+routeTemplate: ./data/component-templates/article.yml
+title: Sitecore Dictionary Service
+---
+
+# Sitecore Dictionary Service
+
+Sitecore Headless Services provides a REST endpoint for retrieving an app-specific translation dictionary, backed by a Sitecore feature called *Dictionary Domains*. 
```

Comment:
Is there a Sitecore doc page we can point to? Sitecore's language translation rules can get pretty complex (with fallback for missing translation). Of course we shouldnt get into the weeds here, but I'm concerned that this is too much of a simplification.

Reply:
Thinking "out loud":  Sitecore has dictionaries, the REST endpoint provides access to them based on **app/site name** and **language**. Language should have a fallback in Sitecore for a bunch of good reasons. 

I propose we improve the presentation by specifying the parameters passed to the REST endpoint and add a note about language fallback with a link to the [language fallback docs](https://doc.sitecore.com/en/developers/101/sitecore-experience-manager/language-fallback.html). I'm not sure though that, for front-end devs, that information is help and not noise...

Reply:
In our meeting, we decided some of these aspects were not relevant to front-end developers and removed potentially confusing / irrelevant info.